src/utils.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { MenuGroup, MenuItem, MenuItemsGroup } from '@grafana/ui';
+import Plotly from 'plotly.js-basic-dist-min';
+import createPlotlyComponent from 'react-plotly.js/factory';
+import { DataFrame } from '@grafana/data';
+import _ from 'lodash';
+
+export const notEmpty = <TValue,>(value: TValue | null | undefined): value is TValue => {
```

Comment:
Is the trailing comma after `<TValue` intended?

Reply:
Yes, it's necessary to have the trailing comma so that it's not parsed as JSX. It's ugly, but the easiest solution.